hangman/lib/hangman/application_supervisor.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Hangman.Application_Supervisor do
+  use Application
+
+  def start_link do
+    import Supervisor.Spec, warn: false
+
+    children = [
+      worker(Hangman.Server, [])
+    ]
+
+    settings = [
+      strategy: :one_for_one,
+      name: Hangman.Server,
+      restart: :permanent
```

Comment:
# -3

I really think this should be transient—if it crashes, we lose the state, so there's not much point in restarting it.